### DISCUSSION TITLE
whiteboard opens images in public links?

### DISCUSSION THREAD

**Discussion by punkyard** [nextcloud/all-in-one#5363] [Questions] [answered]
hi
i have just installed the whiteboard container. next, i have created a public link to a picture. and this link opens whiteboard.. and the picture is not shown. 
before turning this to an issue, i'm submitting this assessment until confirmed by other users.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/whiteboard/issues/191

**Comment by punkyard** [comment]
i didnt think of searching nextcloud server issues. but you did Simon.

you impress me each time.
thanks so much for your investment and accuracy - you teach me a lot

**Reply by szaimen** [reply]
Thanks! Hope that it helps :)

**Reply by punkyard** [reply]
a lot!
